selftests/functional/serial/test_requirements.py

```diff
@@ -76,9 +93,12 @@ def test_c(self):
 '''
 
 
-class BasicTest(TestCaseTmpDir):
+class BasicTest(TestCaseTmpDir, Test):
```

Comment:
It would require a bit more changes, but it would be wise to make this an `avocado.Test` alone (dropping `TestCaseTmpDir`).  I believe it would be simple because `TestCaseTmpDir` is basically about adding what `avocado.Test.workdir` already provides.

Reply:
Sure, I will do that.